### ISSUE TITLE
kramdown version error?

### ISSUE THREAD

**Issue by KnowsCount** [daattali/beautiful-jekyll#756]
```
Bundler could not find compatible versions for gem "kramdown":
  In Gemfile:
    beautiful-jekyll-theme was resolved to 5.0.0, which depends on
      kramdown (~> 2.3.0)

    beautiful-jekyll-theme was resolved to 5.0.0, which depends on
      jekyll (~> 3.8) was resolved to 3.8.7, which depends on
        kramdown (~> 1.14)
```

My kramdown version is 2.3.0: 

```
➜  myBlog git:(gh-pages) ✗ kramdown -v
Resolving dependencies...
2.3.0
```

**Comment by daattali** [comment]
What were the exact steps starting from a new project to reproduce this?

**Comment by KnowsCount** [comment]
why running `bundle exec serve`? I forgot..

**Comment by daattali** [comment]
Closing this issue due to old age and no details, but if you do still have this error then please do open an issue with full reproducible steps

**Comment by Wheest** [comment]
I can confirm that I've got a similar issue.

Steps to get the error:

1. Create new jekyll site: `jekyll new my-site`, `cd my-site`
2. Add the following to `Gemfile`: `gem "beautiful-jekyll-theme", "5.0.0"`.
3. Install packages with `bundle install`

I get the following output:

```sh
Fetching gem metadata from https://rubygems.org/..........
Resolving dependencies...
Bundler could not find compatible versions for gem "kramdown":
  In Gemfile:
    beautiful-jekyll-theme (= 5.0.0) was resolved to 5.0.0, which depends on
      kramdown (~> 2.3.0)

    jekyll (~> 3.8.0) was resolved to 3.8.7, which depends on
      kramdown (~> 1.14)
```

My Gemfile has the following entry for jekyll: `gem "jekyll", "~> 3.8.0" `.

If I change the entry to a more recent version, e.g. `4.1`, I get the following error:

```sh
Fetching gem metadata from https://rubygems.org/..........
Resolving dependencies...
Bundler could not find compatible versions for gem "jekyll":
  In Gemfile:
    jekyll (~> 4.1)

    beautiful-jekyll-theme (= 5.0.0) was resolved to 5.0.0, which depends on
      jekyll (~> 3.8)
```

Which shows that we need `jekyll==3.8` for `beautiful-jekyll-theme`.  The [latest version is 5.0.0](https://rubygems.org/gems/beautiful-jekyll-theme).

**Comment by daattali** [comment]
GitHub Pages seems to still be on jekyll 3.x - I'm not sure what their reasoning is, perhaps the new versions include too many breaking changes? When github pages works with newer version, I'll look into it as well.
